actions/go-mod-tidy/plugin.yaml

```diff
@@ -4,7 +4,7 @@ actions:
     - id: go-mod-tidy
       display_name: Go Mod Tidy
       description: Runs go mod tidy when changes are detected to go.mod.
-      run: ./go-mod-tidy-vendor.sh
-      import_to_global: true
+      runtime: go
+      run: mod tidy
```

Comment:
```suggestion
      run: go mod tidy
```

Reply:
I think you still need this change for it to work

Reply:
I think the intention here is to call go-mod-tidy.sh 

So probably want the run to be 
go-mod-tidy.sh

Reply:
Or this really
      run: ${plugin}/actions/go-mod-tidy/go-mod-tidy.sh